plugins/zoom/plugin.spec.yaml

```diff
@@ -355,3 +361,15 @@ triggers:
         description: User Activity
         type: user_activity
         required: false
+
+tasks:
+  monitor_sign_in_out_activity:
+    title: Monitor Sign In and Out Activity
+    description: Monitor sign in and out activity
+
```

Comment:
can we add `input: null` like we did in other plugins? e.g.  https://github.com/rapid7/insightconnect-plugins/blob/master/plugins/cif/plugin.spec.yaml#L325

Reply:
@gusingh-r7 I'm not sure `input: null` is a pattern we typically use for that - I can't recall ever seeing that in any other plugin 😆

Reply:
There are a bunch of plugins where we use `input: null`. iirc this was for icon-plugin-tool to correctly generate help.md file.